tools/server_lib/test_handler.py

```diff
@@ -4,36 +4,66 @@
 import sys
 import argparse
 import unittest
+import pytest
+import shutil
 
 TOOLS_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 os.chdir(os.path.dirname(TOOLS_DIR))
 
-def handle_input_and_run_tests_for_package(package_name):
-    parser = argparse.ArgumentParser(description="Run tests for the {} package.".format(package_name))
+def handle_input_and_run_tests_for_package(package_name, path_list):
+    parser = argparse.ArgumentParser(description="Run tests for {}.".format(package_name))
     parser.add_argument('--coverage',
                         nargs='?',
                         const=True,
                         default=False,
                         help='compute test coverage (--coverage combine to combine with previous reports)')
+    parser.add_argument('--pytest', '-p',
+                        default=False,
+                        action='store_true',
+                        help="run tests with pytest")
+    parser.add_argument('--verbose', '-v',
+                        default=False,
+                        action='store_true',
+                        help='show verbose output (with pytest)')
     options = parser.parse_args()
 
+    test_session_title = ' Running tests for {} '.format(package_name)
+    header = test_session_title.center(shutil.get_terminal_size().columns, '#')
+    print(header)
```

Comment:
What is this `print(` call for?

Reply:
This outputs a clean header (separator) between each set of tests, using the previous two lines, of the form
```
####...#### Running tests for API ##########...##########
```
centered across the terminal.

Reply:
Oh niice thanks for explaining! :)